dlc-oracle/src/main/scala/org/bitcoins/dlc/oracle/DLCAttestationType.scala

```diff
@@ -0,0 +1,29 @@
+package org.bitcoins.dlc.oracle
+
+import org.bitcoins.crypto.CryptoUtil
+import scodec.bits.ByteVector
+
+sealed trait DLCAttestationType {
```

Comment:
`extends NetworkElement`?

Reply:
This is just an internal type, this won't ever be serialized